doc/source/rllib/package_ref/replay_buffers.rst

```diff
@@ -0,0 +1,132 @@
+.. _replay-buffer-api-reference-docs:
+
+Replay Buffer API
+==================
+
+RLlib comes with a set of extendable replay buffers which are used mostly in Q-Learning algorithms.
+We provide a base :py:class:`~ray.rllib.utils.replay_buffers.replay_buffer.ReplayBuffer` class from which you can build your own buffer.
+Some use cases may require :py:class:`~ray.rllib.utils.replay_buffers.multi_agent_replay_buffer.MultiAgentReplayBuffer`\s.
+You can find buffer types and arguments to modify their behaviour as part of RLlib's default parameters. They are part of
+the ``replay_buffer_config``.
+
+Basic Usage
+===========
+
+You rarely have to instantiate your own replay buffer when running an experiment, but rather configure it as follows.
+The following is from RLlib's examples section and runs the R2D2 algorithm, which runs without `PER <https://arxiv.org/abs/1511.05952>`__ by default, with PER:
```

Comment:
"runs the R2D2 algorithm with PER (which by default it doesn't)." <-- should read more smoothly.